samples/stock/demo/dark-candlestick-and-volume/demo.details

```diff
@@ -0,0 +1,11 @@
+---
+name: 'Dark themed candlestick and volume'
+authors:
+  - Askel Eirik Johansson
+js_wrap: b
+tags:
+  - Highcharts Stock demo
+categories:
+  - General:
```

Comment:
On Daily, we agreed on moving all design-demos to a new category: "Designs". Could you do that too? It seems you also need to create an entry in `demo-config.js` file